docs/configuration.md

```diff
@@ -82,7 +82,8 @@ fladle {
         "- test: ${rootProject.buildDir}/database/module/outputs/apk/database-module-androidTest.apk"
     ]
     autoGoogleLogin = true
-    testShards = 5 //or numUniformShards=5 cannot use both
+    testShards = 5 //or numUniformShards=5 cannot use both. Deprecated use maxTestShards
```

Comment:
this should throw an exception here if both are set, no?

Reply:
This is legacy code. But I think I can logic to verify that as well

Reply:
Ah, sorry I see this is the docs file. My mistake. We should not have `testShards` in the example for the configuration file anymore. Please remove it.
```suggestion
```